docs/10-clusters.md

```diff
@@ -27,6 +27,32 @@ NOTE: The cluster id (--id) can only contain alphanumeric characters including h
 #### Low priority nodes
 You can create your cluster with [low-priority](https://docs.microsoft.com/en-us/azure/batch/batch-low-pri-vms) VMs at an 80% discount by using **--size-low-pri** instead of **--size**. Note that these are great for experimental use, but can be taken away at any time. We recommend against this option when doing long running jobs or for critical workloads.
 
+#### Configuration
+Set your cluster's configuration in a single location using the config/cluster.yaml file. A template with all available configuration settings is provided at config/cluster.yaml.template.
+
+For example, with the following configuration:
+
+```yaml
+# base cluster settings
+base:
```

Comment:
this is not updated right?

also, this section is redundant now that you have your new 12-configuration.md file right?

Reply:
Yes, this is redundant and should be removed.